monarch_gene_mapping/main.py

```diff
@@ -1,12 +1,15 @@
 from os import sep
 import typer
 import pathlib
+
+from curies import get_obo_converter
 from kghub_downloader.download_utils import download_from_yaml
+
 from monarch_gene_mapping.cli_utils import generate_gene_mappings
 from monarch_gene_mapping.uniprot_idmapping_preprocess import filter_uniprot_id_mapping_file
 
 typer_app = typer.Typer()
-
+converter = get_obo_converter()
```

Comment:
since you're using the OBO converter and not a comprehensive one, the standardization operations won't really do anything (there are no synonyms in the OBO converter)

Another possibility is to use the Bioregistry converter, or get an alternative comprehensive converter with different community preferences from the `prefixmaps` package

Reply:
Best option would be for @matentzn to help define a Monarch context in the Bioregistry and then use that :)

Reply:
I am surprised that get_obo_converter() does not get me the synonyms; I thought it would since they are in the epm?

Reply:
So what is the pipeline if I want to standardise all prefixes used to use obo context preferences?

Reply:
Before you do: why does it not use the EPM?

Reply:
It uses http://purl.obolibrary.org/meta/obo_context.jsonld, which is a JSON-LD context, which inherently does not have any synonyms. I guess the confusion here is, why are we not using the Bioregistry's OBO context?

Reply:
> why are we not using the Bioregistry's OBO context?

This Is what I thought we are doing - maybe you mean using the url of the epm? Or just use SSSOM toolkit directly?

Reply:
When you do 

```python

import curies

converter = curies.get_obo_converter()
```

you get http://purl.obolibrary.org/meta/obo_context.jsonld. This is the implementation now. I guess there's probably no harm in updating this to point to the OBO context from the Bioregistry

Reply:
Ahhhhhhhh now I get it!!! OMG I never knew that!

Reply:
https://github.com/monarch-initiative/monarch-gene-mapping/issues/35